src/Dependencies/SignTool/SignTool.csproj

```diff
@@ -6,7 +6,7 @@
     <ProjectTypeGuids>{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="RoslynTools.Microsoft.SignTool" Version="0.2.0-beta" />
+    <PackageReference Include="RoslynTools.Microsoft.SignTool" Version="0.3.3-beta" />
```

Comment:
That's the correct version.  See https://dotnet.myget.org/feed/roslyn-tools/package/nuget/RoslynTools.Microsoft.SignTool/0.3.3-beta